src/View/Helper/HtmlHelper.php

```diff
@@ -143,6 +143,28 @@ public function __construct(View $View, array $config = [])
         parent::__construct($View, $config);
         $this->response = $this->_View->response ?: new Response();
     }
+    
+    /**
+     * Missing method handler.
+     *
+     * If you pass no more than two parameters, it tries to generate a html
+     *   tag with the name of the method and works as alias of `tag()`.
+     * @param string $method Method to invoke
+     * @param array $params Params for the method
+     * @return string|void
+     */
+    public function __call($method, $params)
+    {
+        if (count($params) <= 2) {
```

Comment:
Without a whitelist of allowed tags we won't be able to easily add methods and be confident that we aren't breaking an existing application. I think a whitelist will be necessary.